### ISSUE TITLE
remote connections hyper-v ctrl alt delete ctrl alt end fails

### ISSUE THREAD

**Issue by nvdafan** [NVDARemote/NVDARemote#147]
Hi.
I have been trying to connect to a couple of machines which are Hyper-V virtualized.  When I try, and use the ctrl+alt+delete combination key from within NVDA it says it is unavailable and does not work.  Hyper-V also has a key ctrl+alt+end to get  the secure login screen, this keyboard shortcut also fails to pass through to the machine.  I have the NVDA remote addon copied to the login sessions as directed in the documentation, and if we push ctrl+alt+end on the virtual machine manager directly and go back to the remote session it is possible to enter credentials and move on.
What would be the best way for me to fix this problem?

**Comment by msuzu15** [comment]
Hi, 

I'm not familiar with Hyper V but I'm using VMWare on my local computer.
In case of VMWare, use Ctrl+Alt+Insert instead of Ctrl+Alt+Delete.

By the way, I'm also using virtual machine on Microsoft Azure and I guess it is Hyper-V virtual machine.
I'm using Remote Desktop connection to use virtual machine on Microsoft Azure then Ctrl+Alt+End works in this case.
(where OS on virtual machine is Windows Server 2012 and 2016 and OS on my local computer is Windows 7.)
